server/src/main/java/org/apache/iotdb/db/engine/storagegroup/timeindex/ITimeIndex.java

```diff
@@ -76,14 +76,6 @@
   /** @return Calculate file index ram size */
   long calculateRamSize();
 
-  /**
-   * Calculate file index ram increment when insert data in TsFileProcessor
-   *
-   * @param deviceToBeChecked device to be checked
-   * @return ramIncrement
-   */
-  long estimateRamIncrement(String deviceToBeChecked);
```

Comment:
Why do not count the memory consumed by index?

Reply:
This part of memory is quite small. We can ignore it. 
By the way, it's more necessary to control the memory of sealed TsFileResource....